tests/Feature/Nova/ProductModelTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Products\Tests\Feature\Nova\ProductModelTest;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tipoff\Products\Models\Product;
+use Tipoff\Products\Tests\TestCase;
+
+class ProductModelTest extends TestCase
+{
+    use DatabaseTransactions;
+    use WithFaker;
+
+    /** @test */
+    public function create()
+    {
+        $model = Product::factory()->create();
+        $this->assertNotNull($model);
+    }
+}
```

Comment:
Thanks for creating this test! The only change is to put it in a different folder.

Reply:
It already exists in the correct folder:

https://github.com/tipoff/products/blob/main/tests/Unit/Models/ProductModelTest.php

Reply:
Ahh of course